src/models/User.ts

```diff
@@ -0,0 +1,13 @@
+import * as mongoose from 'mongoose';
+
+export interface IUser {
```

Comment:
In Gitlab repository we have something like:

```typescript
export interface UserModel extends mongoose.Document {
  loginName: string;
  password: string;
}
```

But if we are export `User` and `UserModel` I think we need it separately.